internal/venice-common/src/main/java/com/linkedin/venice/meta/Version.java

```diff
@@ -98,11 +98,14 @@ public static PushType valueOf(int value) {
 
   void setCompressionStrategy(CompressionStrategy compressionStrategy);
 
-  boolean isLeaderFollowerModelEnabled();
+  default boolean isLeaderFollowerModelEnabled() {
+    return true;
+  }
 
   boolean isNativeReplicationEnabled();
 
-  void setLeaderFollowerModelEnabled(boolean leaderFollowerModelEnabled);
+  default void setLeaderFollowerModelEnabled(boolean leaderFollowerModelEnabled) {
+  }
```

Comment:
Same about the `@deprecated` annotation to the LF checks here

Reply:
Done